modules/apps/segments/segments-web/src/main/resources/META-INF/resources/js/components/criteria_builder/CriteriaRowReadable.es.js

```diff
@@ -0,0 +1,207 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+import {PropTypes} from 'prop-types';
+import React, {Component} from 'react';
+
+import {
+	DATE_OPERATORS,
+	NOT_OPERATORS2,
```

Comment:
Should we rename it HAS_OPERATORS?

Reply:
That change is in the following commit. I can edit the commit to include it in this one.

Reply:
Moved to the correct commit